src/messages/direct.rs

```diff
@@ -26,12 +26,10 @@ use std::{
 
 /// Direct message content.
 #[derive(Eq, PartialEq, Serialize, Deserialize)]
-// FIXME - See https://maidsafe.atlassian.net/browse/MAID-2026 for info on removing this exclusion.
-#[allow(clippy::large_enum_variant)]
 pub enum DirectMessage {
     /// Sent from members of a section or group message's source authority to the first hop. The
     /// message will only be relayed once enough signatures have been accumulated.
-    MessageSignature(SignedRoutingMessage),
+    MessageSignature(Box<SignedRoutingMessage>),
```

Comment:
I'm not clear that boxing all those messages make sense. It seem to me that the large_enum_variant warning is not good here and probably should continue to be silenced for that enum (i.e there are some very small variants like ParsecPoke, but most are bigger.

But if we want to remove all of them and be consistent I guess it make sense.